docs/cli/README.md

```diff
@@ -0,0 +1,482 @@
+  altis-cli stack backups [stack]      List backups for the stack.
+  altis-cli stack deploy [stack]       Deploy a given stack.
+  altis-cli stack info [stack]         Get information for a stack.
+  altis-cli stack list                 List stacks available in our hosting.
+  altis-cli stack local-setup [stack]  Setup an existing Altis stack locally.
+  altis-cli stack php-logs [stack]     Show PHP logs for a stack.
+  altis-cli stack scp <src> <dest>     Copy a file to/from a stack. Remote
+                                       src/dest are determined by a colon in the
+                                       input, e.g. stack:/usr/src/app.
+  altis-cli stack ssh [stack]          SSH into a stack.
+```
+
+### Stack backup
+
+The `stack backup` command is used to create and download backups for your application. Note the default is to include both database
+and downloads in the backup.
+
+```sh
+altis-cli stack backup [stack]
+
+Create a new backup for the stack.
+
+Options:
+  --version   Show version number                                      [boolean]
+  --help      Show help                                                [boolean]
+  --uploads   Include uploads in the backup.
+              (--no-uploads to disable)                [boolean] [default: true]
+  --database  Include database in the backup.
+              (--no-database to disable)               [boolean] [default: true]
+  --resume    Log ID for resuming an existing deploy.                   [string]
+  --debug     Enable internal debugging information.  [boolean] [default: false]
+```
+
+For example, to create a database only backup, you would run the following command:
+
+```sh
+bin/altis-cli stack backup --database --no-downloads platform-test
+
+Starting backup for platform-test...
+(node:72569) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
+(Use `node --trace-deprecation ...` to show where the warning was created)
+Backup started, resume later with...
+  hm stack backup platform-test --resume platform-test/backups:2025-02-10-11-33-07
+
+Connected!
+Starting new backup.
+Testing databases connection.
+Running mysqldump. Database size: 917mb
+→ mysqldump: [Warning] Using a password on the command line interface can be insecure.
+→ -- Connecting to platform-test-89c846f8.cnwdi9lvvkax.eu-west-1.rds.amazonaws.com...
+→ -- Retrieving table structure for table wp_10_commentmeta...
+→ -- Sending SELECT query...
+
+...
+
+→ -- Retrieving rows...
+→ -- Disconnecting from platform-test-89c846f8.cnwdi9lvvkax.eu-west-1.rds.amazonaws.com...
+Writing tar file.
+Uploading backup to S3. (111.13 MB)
+Writing manifest file.
+✔ Complete!
+```
+
+If a backup is going to take a long time and in case it may time out, the command outputs the resume command to use to resume
+the same backup.
+
+```sh
+Backup started, resume later with...
+  hm stack backup platform-test --resume platform-test/backups:2025-02-10-11-33-07
```

Comment:
```suggestion
  altis-cli stack backup platform-test --resume platform-test/backups:2025-02-10-11-33-07
```

Reply:
This is real output from thae app. But I've corrected for now, and I'll raise an issue about a number of places that still mention 'hm' and 'hm-cli'.

Reply:
oh! yeah need to fix that :D